hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/DecodingFacade.java

```diff
@@ -136,6 +142,61 @@
 	private static final Bytes ERC_TRANSFER_FROM_SELECTOR = Bytes.wrap(ERC_TRANSFER_FROM_FUNCTION.selector());
 	private static final ABIType<Tuple> ERC_TRANSFER_FROM_DECODER = TypeFactory.create("(bytes32,bytes32,uint256)");
 
+	/* --- Token Create Structs --- */
+	private static final String KEY_VALUE = "(bool,address,bytes,bytes,address)";
+	private static final String KEY_VALUE_DECODER = "(bool,bytes32,bytes,bytes,bytes32)";
+
+	private static final String TOKEN_KEY = "(uint256," + KEY_VALUE + ")";
```

Comment:
Doesn't need to be addressed in this PR, but imo it's a bit wasteful to use a `uint256` -> `BigInteger` to represent what is actually one `byte` of bit masks. 😁